shaders/src/getters.vs

```diff
@@ -29,6 +29,15 @@ int getVertexIndex() {
 #endif
 }
 
+/** @public-api */
+int getInstanceId() {
```

Comment:
Maybe rename to `getInstanceIndex` to be consistent with `getVertexIndex` above?